Election update blog 2026 (#1057)
AlmaLinux/almalinux.org: content/blog/2026-06-08-new-bylaws-and-2026-election.md

The result of all of this work is a more stable organization by overlapping board terms, and easier and more consistent prcesses for us to follow.

## What changed, and why

The goal of this rework was simple: make our elections predictable, fair, and easy to participate in. Three changes do most of that work.

**Elections now happen ever year.** The board has been divided into four cohorts (A, B, C, and D). Each cohort serves a four-year term, and one cohort comes up for election each year. Instead of large, irregular elections, you can count on a single, smaller election every year, at roughly the same time. That makes things easier for you to follow, easier for candidates to plan around, and easier for us to run consistently.

**Voting is now ranked choice.** Rather than picking a single name, you'll rank the candidates in your order of preference. It's a fairer reflection of what the membership actually wants, especially when there are more good candidates than open seats.

> **An independent Election Committee runs the election.** This is how we've been doing things already, but now it's codified in the byloaws: A committee is appointed by the board to run the election, and is made up of members in good standing. Its members can't be nominated in the election they're running.

Reviewer: ```suggestion
**An independent Election Committee runs the election.** This is how we've been doing things already, but now it's codified in the bylaws: A committee is appointed by the board to run the election, and is made up of members in good standing. Its members can't be nominated in the election they're running.
```

The rest of the structure is unchanged from what you'd expect. All six membership classes vote together, with votes weighted by membership level — Platinum 50, Gold 15, Ruby 5, Silver 5, and Individual and Mirror members 1 each. You can read the full details in [the bylaws](/p/foundation-bylaws/).

## The 2026 election

The 2026 election is the first under the new bylaws, and it covers **Cohort A** — these seats were recently held by Jun Yoshida and Jesse Asklund. Jun and Jesse have both already stepped down, so this election will be filling their positions. Here's the timeline:

| Milestone                                   | Date               |
| ------------------------------------------- | ------------------ |

> | Voter eligibility cutoff                    | **July 15, 2026**  |

Reviewer: ```suggestion
| Voter eligibility cutoff                    | **July 16, 2026**  |
```

Reviewer: Cutoff is the 16th

| Nominations open                            | **August 3, 2026** |
| Nominations close, candidate list published | September 8, 2026  |
| Voting opens                                | September 14, 2026 |
| Voting closes                               | October 2, 2026    |
| Results announced                           | by October 9, 2026 |

Going forward, you can expect this same shape every year: eligibility set in mid-July, nominations into August, and voting in September.

## How to get ready

There are two dates worth putting on your calendar right now.

> **July 15 — be a member if you want to vote.** Only members in good standing as of the eligibility cutoff can vote in this year's election. If you've been meaning to join, or your membership has lapsed, now is the time to take care of it. You can [find the membership level that fits you and apply here](/members/). As a reminder: individual membership is free.

Reviewer: Add something about membership requests being reviewed by the membership committee, so don't leave it for the last day?

Reviewer: Also "be a member if you want to vote or run for a board seat"?

Reviewer: ```suggestion
**July 16 — be a member if you want to vote or run for election.** Only members in good standing as of the eligibility cutoff can vote in this year's election. If you've been meaning to join, or your membership has lapsed, now is the time to take care of it. You can [find the membership level that fits you and apply here](/members/). As a reminder: individual membership is free. Membership requests are reviewed by the Membership Committee and approval may take several days, so if you want to be eligible by the deadline, please apply early.
```

"several days" may be too optimistic, feel free to change it.

**August 3 — nominations open.** If you've thought about helping steer the Foundation, or you know someone who'd serve the community well, start thinking about it now. Individual and Mirror members can each nominate one candidate other than themselves, and Sponsor members can nominate a representative. We'll share the full nomination details when the window opens.

## Questions are welcome

> This is a real improvement to how we govern ourselves, and I'd much rather answer your questions than have you guess. The best place to ask is the ~foundation room on [chat.almalinux.org](https://chat.almalinux.org), or you can email me directly.

Reviewer: ```suggestion
This is a real improvement to how we govern ourselves, and I'd much rather answer your questions than have you guess. The best place to ask is the [~foundation](https://chat.almalinux.org/almalinux/channels/foundation) room on [chat.almalinux.org](https://chat.almalinux.org), or you can email me directly.
```

"we'd much rather"?

Thank you, as always, for the trust you put in us and for everything you do for each other. I'm looking forward to a clean, straightforward election this year.

## Want to help?

> As a note: if you are intersted in joining the election committee, please let me know! It's a very low-lift process (probably 1 - 3 hours each week during the election window at most). That's especially true with the introduction of our new Accounts software, but your priticipation increases the accountability we have, and validates the trust you're putting in us.

Reviewer: ```suggestion
As a note: if you are intersted in joining the election committee, please let me know! It's a very low-lift process (probably 1 - 3 hours each week during the election window at most). That's especially true with the [introduction of our new Accounts software](/blog/2026-03-12-introducing-almalinux-astra/), but your participation increases the accountability we have, and validates the trust you're putting in us.
```
